### DISCUSSION TITLE
window.open

### DISCUSSION THREAD

**Discussion by jleviaguirre** [TIBCOSoftware/spotfire-mods#112] [Q&A] [answered]
Is there a way to allow window.open? A customer is trying to download the marked rows from context menu. The sandbox environment makes it challenging for this type of use case. Any work arounds, ideas or solutions for it? The original question was posted [here](https://community.tibco.com/s/question/0D54z00008vJlbiCAC/implementation-of-oncontextmenu)

**Comment by objerke** [comment, marked as answer]
This is currently not supported by the Mods API. It is possible to download the data from the marked rows a mod via the "Data to file" option. The only caveat is that the mod needs to enable `tooltip` property in the `dataViewDefinition` in the (mod-manifest.json)[https://tibcosoftware.github.io/spotfire-mods/mod-schema/index.html#:~:text=for%20this%20axis.-,tooltip,-%3A%20object%20%5Bobject]. This was added in version 1.3 of the Mods API.

![image](https://user-images.githubusercontent.com/8665943/234779330-d74b821d-91cd-43ab-b728-6085df3fd689.png)
![image](https://user-images.githubusercontent.com/8665943/234779464-1b3faaf5-9142-47c6-aa8b-af00bb8367eb.png)
